src/proto/grpc/testing/control.proto

```diff
@@ -34,20 +34,16 @@ import "src/proto/grpc/testing/stats.proto";
 
 package grpc.testing;
 
-enum ClientType {
-  SYNC_CLIENT = 0;
-  ASYNC_CLIENT = 1;
-}
-
-enum ServerType {
-  SYNC_SERVER = 0;
-  ASYNC_SERVER = 1;
-  ASYNC_GENERIC_SERVER = 2;
-}
-
 enum RpcType {
-  UNARY = 0;
-  STREAMING = 1;
+  // First there are ping-pong RPCs in which the client sends a request and the
+  // server sends a response
+  UNARY = 0; // unary RPC, ping-pong by nature
+  STREAMING_PING_PONG = 1; // streaming ping-pong RPCs
```

Comment:
considering we are already running the tests on jenkins in multiple languages, changing the naming here will probably require a bit more coordination between language implementations.
Also, we will need to test that all the languages work before we submit this change.